### DISCUSSION TITLE
Do I need to reapply the playbook after installing driver updates through Windows Update?

### DISCUSSION THREAD

**Discussion by zenwav** [Atlas-OS/Atlas#1323] [Help & Support] [answered]
Does the same apply for security patches and feature updates? When exactly should I rerun my playbook after Windows updates?

**Comment by Ast3risk-ops** [comment]
> Does the same apply for security patches and feature updates? When exactly should I rerun my playbook after Windows updates?

Windows updates are supported normally. You never need to reapply the playbook.

**Reply by zenwav** [reply]
Wouldn't a feature update require running the playbook again, since it might make changes that undo AtlasOS's previous system modifications?

**Reply by Ast3risk-ops** [reply, marked as answer]
> Wouldn't a feature update require running the playbook again, since it might make changes that undo AtlasOS's previous system modifications?

Atlas holds back problematic feature updates (you won't see them) and has a few mechanisms in place for tweak preservation.
